perf-cluster-operation/pkg/benchmark.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strings"
+
+	yaml "gopkg.in/yaml.v2"
+	"knative.dev/test-infra/shared/common"
+)
+
+const (
+	// clusterConfigFile is the config file needs to be put under the benchmark folder if we want to config
+	// the cluster that runs the benchmark, it must follow the scheme defined as GKECluster here.
+	clusterConfigFile = "cluster.yaml"
```

Comment:
Do you mean the file name or the yaml scheme as well?

Reply:
I mean the location of this file, can consumer of this package provide an absolute path of the yaml?

Reply:
I think it would be better to have a common standard for organizing the benchmark folder, and add it to the instruction of how to write a new benchmark. So I would prefer to only define it here.